easybuild/easyconfigs/r/REViewer/REViewer-0.2.7-foss-2024a.eb

```
@@ -0,0 +1,52 @@
+easyblock = 'CMakeMake'
+
+name = 'REViewer'
+version = '0.2.7'
+toolchain = {'name': 'foss', 'version': '2024a'}
+toolchainopts = {'pic': True, 'cstd': 'c++14'}
+
+
+homepage = 'https://github.com/Illumina/REViewer'
+description = """REViewer is a tool for visualizing alignments of reads in regions containing tandem repeats. REViewer
+ requires a BAMlet with graph-realigned reads generated by ExpansionHunter and the corresponding variant catalog."""
+source_urls = ['https://github.com/Illumina/REViewer/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'GraphCoordinates.cpp.patch',
+    ('graph-tools.CMakeLists.txt.patch', 'reviewer/thirdparty/graph-tools'),
+    ('reviewer.CMakeLists.txt.patch', 'reviewer')
```

We usually give the version in the patch
```suggestion
    'REViewer-0.2.7_GraphCoordinates_cpp.patch',
    ('REViewer-0.2.7_graph-tools_CMakeLists.patch', 'reviewer/thirdparty/graph-tools'),
    ('REViewer-0.2.7_reviewer_CMakeLists.txt.patch', 'reviewer')
```